crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -2475,6 +2490,23 @@ mod tests {
         assert!(world.get::<Z>(e).is_some());
     }
 
+    #[test]
+    fn unregister_system_cached_commands() {
+        let mut world = World::default();
+        let mut queue = CommandQueue::default();
+
+        fn nothing() {}
+        assert!(world.iter_resources().count() == 0);
+        let _id = world.register_system_cached(nothing);
```

Comment:
Could actually use this system ID to check that the entity it points to (`SystemId::entity()`) exists, and later that the entity no longer exists after unregister.

Reply:
Much better, thanks!